templates/nextcloud-setup.sh

```diff
@@ -39,8 +43,15 @@ elif [ "$INSTALLED" == "null" ]; then
     sleep 6
     exit 1
 fi
-
-
+(
+set +x
+export OC_PASS="$UPLOADS_USER_PASS"
+php occ user:resetpassword --password-from-env uploads
+export OC_PASS="$NEXTCLOUD_ADMIN_PASSWORD"
+php occ user:resetpassword --password-from-env $NEXTCLOUD_ADMIN_USER
+)
+echo "uploads and admin password set
+"
```

Comment:
I've set the environment variable with `export` instead of using @gabriel-v s command with `sudo -u www-data` because www-data is not in the sudoers file.

Reply:
Yes, we already have the user switch [here].